pom.xml

```diff
@@ -790,57 +816,74 @@
         <dependency>
             <groupId>com.github.tomakehurst</groupId>
             <artifactId>wiremock</artifactId>
-            <version>2.7.1</version>
+            <version>2.8.0</version>
             <scope>test</scope>
             <exclusions>
-                <!-- Upper bound dependency fix: later version comes from org.jenkins-ci.main:jenkins-test-harness -->
-                <exclusion>
-                    <groupId>org.eclipse.jetty</groupId>
-                    <artifactId>jetty-webapp</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.eclipse.jetty</groupId>
-                    <artifactId>jetty-server</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.eclipse.jetty</groupId>
-                    <artifactId>jetty-servlet</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.eclipse.jetty</groupId>
-                    <artifactId>jetty-http</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.eclipse.jetty</groupId>
-                    <artifactId>jetty-util</artifactId>
-                </exclusion>
-                <exclusion>
-                    <groupId>org.eclipse.jetty</groupId>
-                    <artifactId>jetty-io</artifactId>
-                </exclusion>
                 <!--Upper bound fix, pick one from the core -->
                 <exclusion>
                     <groupId>org.ow2.asm</groupId>
                     <artifactId>asm</artifactId>
                 </exclusion>
             </exclusions>
         </dependency>
+        <!--
+            wiremock jetty, depends on 9.2.22.v20170606, jenkins-test-harness on jetty 9.4.5.
+
+            jetty-util 9.4.5 breaks backward compatibility, where
+
+            SslContextFactory.selectCipherSuites([Ljava/lang/String;[Ljava/lang/String;)[Ljava/lang/String;
```

Comment:
https://github.com/tomakehurst/wiremock/issues/551#issuecomment-268647572

use wiremock-standalone that shades jetty...  did you try this?

Reply:
@jtnord thanks for the pointer, no I haven't. Happy to try as it will clean all that work around.